### ISSUE TITLE
Trying to launch STL from Steam no longer works

### ISSUE THREAD

**Issue by hazelnot** [sonic2kk/steamtinkerlaunch#410]
I can't seem to be able to launch STL from Steam anymore. The app menu shortcut still works, but that doesn't help me very much in this situation.

It seems to not run at all, as no log has been generated. The last log files I have are from when I launched the standalone version: [31337.log](https://github.com/frostworx/steamtinkerlaunch/files/8010172/31337.log), [steamtinkerlaunch.log](https://github.com/frostworx/steamtinkerlaunch/files/8010173/steamtinkerlaunch.log)
 
I'm on v9.0.1, on Arch Linux

**Comment by frostworx** [comment]
When steamtinkerlaunch still works standalone, this is no steamtinkerlaunch issue.
Of course Valve can change Steam and break steamtinkerlaunch anytime, but for me, it still works as expected, so I assume this is a local issue on your machine.

You might want to restart steam/your machine to check if that changes anything.

**Comment by hazelnot** [comment]
This started happening months ago though, restarting stuff wouldn't change anything. Not sure what else I could try.

**Comment by frostworx** [comment]
Don't you think that this would be already a known issue if steamtinkerlaunch broke generally "months ago"?

**Comment by hazelnot** [comment]
Ok, but any idea how I could get it working again? Cause I have absolutely no clue what's going on here.

**Comment by frostworx** [comment]
I can only guess here apparently.
`stl` was renamed recently to `steamtinkerlaunch`, which you likely recognized.
There were several migration functions to make the migration easier for the user, but, depending on the time when you used steamtinkerlaunch you might have missed the helper (iirc I removed it shortly after the transition went into a release).
TL;DR:
make sure your files in `~/.steam/steam/compatibilitytools.d/SteamTinkerLaunch/` are up to date _(so the symlink to the program is valid and the vdf files use `steamtinkerlaunch` and not `stl`)_

or easier, simply recreate it.
With some luck that was already everything.

**Comment by frostworx** [comment]
so?

**Comment by hazelnot** [comment]
Sorry, I forgot to try this, and I don't have access to my desktop atm. I started using this in November though, and I'm pretty sure the files/folders are already called `steamtinkerlaunch`, but I'll double-check when I get home

**Comment by frostworx** [comment]
¯\\_(ツ)_/¯

**Comment by hazelnot** [comment]
Sorry my keyboard broke and I can't check lmao. If I remember to I'll update this

Just FYI though, "not my problem, you're doing it wrong" is a really crappy way of responding to issues

**Comment by frostworx** [comment]
wtf, in fact I was trying to help you, but you never ever had the decency to even say thank you. so please don't open any issues anymore. thank you!

**Comment by hazelnot** [comment]
Say thank you for what, for replying with stuff like "Don't you think that this would be already a known issue if steamtinkerlaunch broke generally "months ago"?"?

Your "trying to help" reads a lot like most people's passive-aggressiveness and being dismissive.
